File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        set<int> answer;
        for(int i = 0; i < nums.size(); i++){
            if(!answer.count(nums[i])){
                answer.insert(nums[i]);
            }
            else{
                return true;
            }
        }
        return false;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        tracker = set()
        for n in nums:
            tracker.add(n)

        answer = 0
        count = 0
        for n in nums:
            if (n - 1) in tracker:
                continue
            else:
                for i in range(len(nums)):
                    if (n + i) in tracker:
                        count += 1
                    else:
                        break
                answer = max(count, answer)
            count = 0
        return answer